# BC7 linear texels are decoded with the sRGB transfer function.
--- a/src/mullion/mullion_format.cpp
+++ b/src/mullion/mullion_format.cpp
@@ -1112,7 +1112,7 @@
     break;
   }
   case DXGI_FORMAT_BC7_UNORM: {
-    description.PixelFormat = WMTPixelFormatBC7_RGBAUnorm;
+    description.PixelFormat = WMTPixelFormatBC7_RGBAUnorm_sRGB;
     description.Flag = MTL_DXGI_FORMAT_BC;
     description.BlockSize = 16;
     break;
